models/core/blocks.sql

```diff
@@ -2,30 +2,59 @@
     config(
         materialized='incremental',
         unique_key='block_id',
+        incremental_strategy = 'delete+insert',
         tags=['core'],
         cluster_by=['block_timestamp']
     )
 }}
 
--- WIP -- below serves as example
-with base_blocks as (
+with
 
-    select * from {{ deduped_blocks("near_blocks") }}
+base_blocks as (
+
+    select
+        *
+    from {{ ref("stg_blocks") }}
+    where {{ incremental_load_filter("block_timestamp") }}
 
 ),
 final as (
+
     select
-        block_id,
-        block_timestamp,
+        header:height::integer as block_id,
```

Comment:
reckon we can directly use the `block_id` here
```suggestion
        block_id,
```

Reply:
![image](https://user-images.githubusercontent.com/9086733/152624361-caf96637-f482-4537-93b6-b1e251c91501.png)

Reply:
I've checked the data in this 1 row, seems like its the 20365766's data.